src/software/robot_diagnostics/widgets/dribbler.cpp

```diff
@@ -0,0 +1,14 @@
+#include "software/robot_diagnostics/widgets/dribbler.h"
+
+void setupDribbler(Ui::AutoGeneratedMainWidget *widget, double &dribbler_power)
+{
+    setupSliderLineEdit(widget->lineEdit_dribbler_power, widget->slider_dribbler_power,
+                        dribbler_power, 0.0, 99.0, 1.0);
+
+    auto dribbler_run_pushed = []() {
+        // TODO: send some proto
```

Comment:
Can we link to #1229 here? Same goes for the similar `TODO`'s in other files.

Reply:
done